tests/conftest.py

```diff
@@ -25,6 +25,7 @@ def config(kube_ns):
     """
     cfg = Config()
     cfg.KubeSpawner.namespace = kube_ns
+    cfg.KubeSpawner.start_timeout = 120
```

Comment:
I got several time random timeouts with checks so I decided to increase it to solve this issue and make tests more stable :-/ 
I can try to revert this if you want.

Reply:
I think the timeout errors will happen no matter what, and I'm trying to address them in another PR. So, due to this, let's remove it so it isn't a confusion about it later that this may be helpful when it just delays an inevitable failure.

For reference, here is a travis build where the timeouts are showing: https://travis-ci.org/github/jupyterhub/kubespawner/builds/737533326, but the issue I think is tracked in: https://github.com/jupyterhub/jupyterhub/issues/3222

Reply:
thanks for the link, I just reverted the start_timeout, hope the checks will pass